sgf-viewer/src/components/NavBar.svelte

```diff
@@ -1,185 +1,85 @@
-<div class="nav-bar-container">
-    {#if innerWidth > menuNavigationWidth}
-        <div class="nav-bar" bind:this={navBarElem}>
-            <!-- <div class="icons flex-grow-symmetric">
-                <a
-                    class="icon-link"
-                    on:click={() => {
-                        currentPath = "";
-                        history.pushState({}, "", currentPath);
-                    }}
-                    href="#"
-                >
-                    <TiHome />
-                </a>
-            </div> -->
-            <div class="pages">
-                {#each pagesPaths as page}
-                    <NavButtons
-                        label={pages[page]["title"]}
-                        tabsId="tabs"
-                        selected={page === currentPath}
-                        href="#contents"
-                        onClick={() => {
-                            currentPath = page;
-                            history.pushState({}, "", currentPath + "#contents");
-                        }}
-                    />
-                {/each}
-            </div>
-            <!-- <div class="flex-grow-symmetric" /> -->
-        </div>
-    {:else}
-        <div class="menu-bar" bind:this={navBarElem}>
-            <!-- <div class="flex-grow-symmetric" /> -->
-            <!-- <p class="menu-selected">{pages[currentPath]["title"]}</p> -->
-            <!-- <div class="menu-burger flex-grow-symmetric">
-                <div >
-                    <a on:click={() => menuOpen = !menuOpen}>
-                        <IoIosMenu />
-                    </a>
-                </div>
-            </div> -->
-            <!-- {#if menuOpen} -->
-            <select
-                bind:value={currentPath}
-                on:change={() => {
-                    history.pushState({}, "", currentPath);
-                    location.replace("#contents");
-                }}
-            >
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <div class="container-fluid">
+        <a class="navbar-brand" href="https://far.ai">
+            <img src="/images/far-logo.svg" alt="FAR logo" width="24" height="24" class="d-inline-block align-text-top">
+            FAR
+        </a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent" bind:this={navbarSupportedContent}>
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
                 {#each pagesPaths as page}
-                    <option value={page}>{pages[page]["title"]}</option>
+                    <li class="nav-item">
+                        <a class="nav-link"
+                           class:active={page == currentPath}
```

Comment:
I find the active vs inactive quite subtle (ever so slight change in font color), can we make this stand out more?

E.g. we could use #009999 "A medium teal used for accents, complementing the main logo color." from the [brand guidelines](https://docs.google.com/document/d/1RYEhcJwhDXe4nu0bhqukuh_LPbHWbmYcVK38mC4BnjY/edit)

Reply:
I used #226666 instead since #009999 is now used for links, but #226666 is also a bit subtle